docs/checkers/comments.md

```diff
@@ -0,0 +1,19 @@
+Detects uninformative comments
```

Comment:
Detects comments that aim to silence `go lint` complaints about exported symbol not having a doc-comment.
(Also note that period is missing.)